modules/configuration/remote_linux/base_config.lua

```diff
@@ -0,0 +1,16 @@
+--- Module which provides ATF base configuration
+--
+-- *Dependencies:* none
+--
+-- *Globals:* none
+-- @copyright [Ford Motor Company](https://smartdevicelink.com/partners/ford/) and [SmartDeviceLink Consortium](https://smartdevicelink.com/consortium/)
+-- @license <https://github.com/smartdevicelink/sdl_core/blob/master/LICENSE>
+
+local config = { }
+
+--- Define path to SDL binary
+-- Example: "/home/user/sdl_build/bin"
+config.pathToSDL = "/home/developer/sdl/b/dev/p/bin"
+config.pathToSDLPolicyDB = "policy.sqlite"
+
```

Comment:
Should also add
```lua
config.pathToSDLConfig = config.pathToSDL
```
here so it uses the .INI file from the host

Reply:
@ShobhitAd Please see my previous comment